### DISCUSSION TITLE
School updated domain but didn't update the info for GIthub Student

### DISCUSSION THREAD

**Discussion by ghost** [community/community#118901] [GitHub Education]
### Hi everyone,

I was trying to apply for Github Student, I noticed that my school's old email/domain was still registered while the new one wasn't.
How do I get in touch with the right people to get them to update this info?
(old domain: isissmotta.gov.it, new domain: antonioscarpa.edu.it)

Thanks

**Comment by SupunJayaweera** [comment]
Hello there,

It seems like you're trying to apply for the GitHub Student Developer Pack and have noticed that your school's old email/domain is still registered while the new one isn't. Here's how you can get this updated:

1. **Contact GitHub Support**: You can reach out to GitHub Support through the GitHub Support contact form. They are the best point of contact for this kind of issue.

2. **Provide Detailed Information**: When you contact GitHub Support, make sure to provide all the necessary details. This includes your old school domain (`isissmotta.gov.it`), your new school domain (`antonioscarpa.edu.it`), and any other relevant information. This will help them understand and address your issue more effectively.

3. **Follow Up**: If you don't hear back from GitHub Support within a reasonable time frame, don't hesitate to send a follow-up message. They deal with a high volume of queries, so a gentle reminder can help ensure your request doesn't get lost in the shuffle.

Remember, GitHub is there to support you in your learning journey. Don't hesitate to reach out to them with any issues or questions you might have.

Best of luck with your application for the GitHub Student Developer Pack!

Happy coding!

**Comment by samus-aran** [comment]
Hi @ValerioConsoli  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
